addons/calendar/data/mail_template_data.xml

```diff
@@ -90,7 +90,7 @@
                 </t>
                 <t t-if="object.event_id.location">
                     <li>Location: <t t-out="object.event_id.location or ''">Bruxelles</t>
-                        (<a target="_blank" t-attf-href="http://maps.google.com/maps?oi=map&amp;q={{object.event_id.location}}">View Map</a>)
+                        <a target="_blank" t-if="not is_online or object.event_id.location != object.event_id.appointment_type_id.location_id.name" t-attf-href="http://maps.google.com/maps?oi=map&amp;q={{object.event_id.location}}">(View Map)</a>
```

Comment:
Just tested with the new changes and it seems it does not work like I hoped.
So we will need to be a bit more verbose here by doing this:
`not is_online or is_online and object.event_id.location != object.event_id.appointment_type_id.location_id.name`

Reply:
My bad..I should've tested too : /
Will push this change with the other one altogether :+1: